package/yast2-online-update.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-online-update
-Version:        4.0.1
+Version:        4.0.2
```

Comment:
JFYI: For Leap 15.1 all YaST packages should use the 4.1.x version. But we will bump the version later if needed, you can leave that...